subprojects/snapshots/src/main/java/org/gradle/internal/vfs/impl/WatchingVirtualFileSystem.java

```diff
@@ -117,6 +118,11 @@ public void handleLostState() {
             });
             long endTime = System.currentTimeMillis() - startTime;
             LOGGER.warn("Spent {} ms registering watches for file system events", endTime);
+        } catch (UnableToStartWatchingException ex) {
```

Comment:
Should we call this `WatchingNotSupportedException`?

Reply:
Renamed.